content/en/guides/os-commercial-ecosystem.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Creating an Open Source Commercial Ecosystem
+---
+
+An open source project is successful when it has a thriving ecosystem of companies and products built around it.  So what exactly is an open source commercial ecosystem and how can organizations help create and support sustainable open source projects? There are concrete ways to create confidence in a project’s long-term viability, encourage companies to create commercial products and services on top of open source projects, and re-invest in them with contributions, content, and other resources. These practices for enabling ecosystem development help set your projects up for success.
+
+&nbsp;
```

Comment:
These `&nbsp;`s aren't needed in markdown. Please remove them wherever they occur in the document.

Reply:
What other way we can follow to add spaces in a markdown document?

Reply:
Since this does not affect the content of this guide and it's been reviewed and opened for a long time, my suggestion is to merge this guide the way it is and we can later delete `&nbsp;` in a second iteration.